README.md

```diff
@@ -233,6 +233,19 @@ string transformedAssetUrl = builder.WithFocalPointCrop(560, 515, 2)
 
 For list of supported transformations and more information visit the Kentico Delivery API reference at <https://developer.kenticocloud.com/v1/reference?#image-transformation>.
 
+## Resilience capabilities
```

Comment:
this whole chapter could use some copyediting. could you please create a task in the CTC backlog?

Reply:
CTC-256